### PR TITLE
New version: KristenMcWilliam.Nyrna version 2.15.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115076&drop=dogfoodAlpha

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
Looks like this package writes `2.9.0` to control panel

**Comment by stephengillie** [comment]
image

File: manifests/k/KristenMcWilliam/Nyrna/2.15.0/KristenMcWilliam.Nyrna.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-07-21
```
Comment (stephengillie) [inline manifests/k/KristenMcWilliam/Nyrna/2.15.0/KristenMcWilliam.Nyrna.installer.yaml:15]:
```suggestion
ReleaseDate: 2023-07-21
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**